client/asset/eth/nodeclient_harness_test.go

```diff
@@ -682,8 +689,12 @@ func testInitiate(t *testing.T) {
 			t.Fatalf("%s: balance error: %v", test.name, err)
 		}
 
+		gasPrice, err := feesAtBlk(ctx, ethClient, receipt.BlockNumber.Int64()-1)
+		if err != nil {
+			t.Fatalf("%s: feesAtBlk error: %v", test.name, err)
+		}
 		bigGasUsed := new(big.Int).SetUint64(receipt.GasUsed)
-		txFee := new(big.Int).Mul(bigGasUsed, baseFee) // DRAFT NOTE: Why don't I need to add the tip?
+		txFee := new(big.Int).Mul(bigGasUsed, gasPrice)
```

Comment:
I think @martonp had explained this to me, but I can't find it now. Everything I read seems to indicate that tx fees should be assessed at `(baseFeeRate + priorityFeeRate) * gas`, but these calculations seem to have no priority fee added.

Reply:
Ah, the `feesAtBlk` helper added in this PR for computing this `gasPrice` value is doing that.

BTW, the computation there is part of my motivation for https://github.com/decred/dcrdex/pull/1356 (so the needed ChainConfig can be loaded by server) and the linked [possible followup change](https://github.com/decred/dcrdex/commit/64a72b48fe3b05fcf8d5f4a7d9296496092967ac#r62517085) that builds on that PR to do the same calculation for `server/asset.(*Backend).FeeRate`, even though it's probably overkill for that need.

And I apologize in advance for the extremely long-winded issue https://github.com/decred/dcrdex/issues/1372.  I'm actually trying to get consensus about these eth fee issues there, with a focus on the prescribed match rates that get recorded and enforce.

Reply:
> the `feesAtBlk` helper added in this PR for computing this `gasPrice` value is doing that

Ah. Derp.